core/state/statedb.go

```diff
@@ -333,17 +333,6 @@ func (s *StateDB) GetStorageProof(a common.Address, key common.Hash) ([][]byte,
 	return proof, err
 }
 
-// GetStorageProofByHash returns the Merkle proof for given storage slot.
-func (s *StateDB) GetStorageProofByHash(a common.Address, key common.Hash) ([][]byte, error) {
```

Comment:
This function seems to have been added here: https://github.com/ethereum/go-ethereum/commit/017831dd5b33a68076aed7c9ff05e62b0dcb5f92#diff-c3757dc9e9d868f63bc84a0cc67159c1d5c22cc5d8c9468757098f0492e0658cR340
Not sure about the context or use

Reply:
yeah, I have searched the go-ethereum project, 
no context  found to use it